.php-cs-fixer.php-highest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+if (PHP_VERSION_ID <= 80001 || PHP_VERSION_ID >= 80100) {
+    fwrite(STDERR, "PHP CS Fixer's config for PHP-HIGHEST can be executed only on highest supported PHP version - 8.0.*.\n");
+    fwrite(STDERR, "Running it on lower PHP version would prevent calling migration rules.\n");
+    exit(1);
+}
+
+$config = require '.php-cs-fixer.dist.php';
+
+$config->setRules([
+    '@PHP80Migration' => true,
+    '@PHP80Migration:risky' => true,
```

Comment:
moving the configuration to file is potentially allowing us, as 2nd step, to reuse default config + add PHP80Migration* so the 2nd run is not failing due to misstyled code - ref https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5754#issuecomment-912976954